README.md

```diff
@@ -40,6 +40,9 @@ const client = createClient({
     componentVersion: require('./package.json').version, // (optional) automated version reporting
     deviceId: passedDeviceId,
 })
+
+//And at last, we pass this newly created client to the AnalyticsUrlParams since its needed for some functionalities
```

Comment:
```suggestion
// A newly created client is passed to the `AnalyticsUrlParams` to make it fully functional.
```
No need to use this specific suggestion, but my idea is to unify the spacing, punctuation and maybe reduce the number of unnecessary filler words in the comments of the example snippet.

Reply:
Yeap, it sounds better thay way!